sound/soc/bcm/hifiberry_dacplus.c

```diff
@@ -342,6 +398,8 @@ static int snd_rpi_hifiberry_dacplus_probe(struct platform_device *pdev)
 	struct device_node *tpa_node;
 	struct property *tpa_prop;
 	struct of_changeset ocs;
+	int tmp;
+	struct property *pp;
```

Comment:
It's a tiny niggle, but kernel devs like variable declarations in order of decreasing length - it just looks nicer with `int tmp` at the end.

Reply:
ok